app/views/contexts/submission_metadata_context.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+
+from flask_babel import format_datetime, lazy_gettext
+
+from app.libs.utils import convert_tx_id
+
+
+def build_submission_metadata_context(
+    survey_type: str, submitted_at: datetime, tx_id: str
+) -> dict:
+    submitted_on = {
+        "term": lazy_gettext("Submitted on:"),
+        "descriptions": [
+            {
+                "description": lazy_gettext(
+                    "{date} at {time}".format(
+                        date=format_datetime(submitted_at, format="dd LLLL yyyy"),
+                        time=format_datetime(submitted_at, format="HH:mm"),
+                    )
+                )
```

Comment:
The format needs to be called after `lazy_gettext`.
```suggestion
                "description": lazy_gettext("{date} at {time}").format(
                    date=format_datetime(submitted_at, format="dd LLLL yyyy"),
                    time=format_datetime(submitted_at, format="HH:mm"),
                )
```